app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/fragment/EventDetailsFragment.kt

```diff
@@ -130,9 +125,24 @@ class EventDetailsFragment : Fragment(), SwipeRefreshLayout.OnRefreshListener {
         startActivity(intent)
     }
 
-    private fun showTicketCount(count: Int?) {
-        val text = count?.toString() ?: getString(R.string.unknown)
-        remainingTicketsTextView.text = text
+    private fun showTicketCount(status: TicketStatus?) {
+        if(EventHelper.isEventImminent(event)) {
```

Comment:
This looks a lot like the code from below. But I guess its rather too complicated to somehow pass in the correct variables.

Please at least add comments, that those instructions should be kept the same.